fe/fe-core/src/main/java/com/starrocks/connector/PartitionDataInfo.java

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.connector;
+
+public class PartitionDataInfo {
```

Comment:
Is this for all connector types: iceberg/hudi/hive? or only for hive? if it's only for hive, we can add hive prefix?

Reply:
only for hive

Reply:
ok